### PR TITLE
Map the rest of BoatType, as well as some other entity related things.

### PR DESCRIPTION
**Pull request by darkevilmac**

### CODE REVIEW

File: mappings/net/minecraft/block/Block.mapping [hunk 1]
```diff
@@ -125,10 +125,11 @@ CLASS none/akw net/minecraft/block/Block
 		ARG 0 world
 		ARG 1 pos
 		ARG 2 entity
-	METHOD a (Lnone/aiu;Lnone/cn;Lnone/se;F)V
+	METHOD a onEntityLanding (Lnone/aiu;Lnone/cn;Lnone/se;F)V
```
Comment (darkevilmac) [inline mappings/net/minecraft/block/Block.mapping]: onEntityLanding, or onLandedUpon?

Comment (Darkhax) [reply mappings/net/minecraft/block/Block.mapping]: I think `onLandedUpon` would be better.

**Comment by asiekierka** [comment]
LGTM

**Comment by shadowfacts** [comment]
LGTM

**Pull request merged**